gef.py

```diff
@@ -10545,14 +10315,14 @@ def __load_extra_plugins(self) -> int:
             directories = gef.config["gef.extra_plugins_dir"]
             if directories:
                 for directory in directories.split(";"):
-                    directory = os.path.realpath(os.path.expanduser(directory))
-                    if os.path.isdir(directory):
-                        sys.path.append(directory)
-                        for fname in os.listdir(directory):
-                            if not fname.endswith(".py"): continue
-                            fpath = f"{directory}/{fname}"
-                            if os.path.isfile(fpath):
-                                gdb.execute(f"source {fpath}")
+                    directory = pathlib.Path(directory).expanduser()
+                    if not directory.is_dir():
+                        continue
+                    for entry in directory.iterdir():
+                        if not entry.is_file(): continue
+                        if entry.suffix != ".py": continue
+                        if entry.name == "__init__.py": continue
```

Comment:
Shouldn't we still run these? Most might be empty but what if people put stuff in theirs?

Reply:
I'm not sure about that. This is supposed to be where `gef-extras/scripts` point to and I must admit I don't do much vetting about what goes in `gef-extras`.  Sourcing `__init__.py` might end up in the repo and breaking stuff for everyone.

The way I use it is by creating "mini-packages" with a top level script that declares the commands and will import the rest. This way I (or anyone actually 😄 ) can create large plugins in multiple files without risking stepping on someone else's toes 
```
gef➤  !ls -l ~/code/gef-extras/scripts
drwxr-xr-x    - hugsy  8 Feb  9:31 foo
`.rwxr-xr-x`   23 hugsy  8 Feb  9:31 foo.py

gef➤  !cat  ~/code/gef-extras/scripts/foo.py
from foo import A
A()

gef➤  !cat  ~/code/gef-extras/scripts/foo/__init__.py
class A:
    def __init__(self):
        print("hello package")
```